### PR TITLE
Apply isort

### PR DESCRIPTION
**Pull request by yannikschaelte**
* Apply isort via pre-commit hook to sort all imports

(i.e. just automatic formatting changes, more or less no actual changes in this pr)

* limit number of function evaluations in tests via #787

### CODE REVIEW

File: pypesto/C.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
+"""
+Constants
+=========
+
+Package-wide consistent constant definitions.
+
+"""
+
+from enum import Enum
```
Comment (FFroehlich) [inline pypesto/C.py:9]: how about turning all of these into proper enums? I think with multiple inheritance from `str` and `Enum` we should maintain backwards compatibility.

Would keep imports smaller and enable better input checking.

**Review by FFroehlich — CHANGES_REQUESTED** [review]
image

can we keep things in separate (minimal) PRs if >100 files are touched? This is an absolute pain to review.

File: pypesto/C.py (hunk 1 above)
Comment (yannikschaelte) [reply pypesto/C.py:9]: What for example would you mean? Regarding brevity of imports, I had in mind anyhow only importing `from pypesto import C` and then using e.g. `C.MODE_FUN`, which should be nice and readable already.

Comment (yannikschaelte) [reply pypesto/C.py:9]: > image
> 
> can we keep things in separate (minimal) PRs if >100 files are touched? This is an absolute pain to review.

Yeah ... the C + refactoring of the ensemble module became necessary along the way, but will try to extract the actual code changes there, to separate from the mere sorting.

**Review by FFroehlich — APPROVED** [review]

**Pull request merged**